unliftio/unliftio.cabal

```diff
@@ -22,6 +22,11 @@ extra-source-files:
     README.md
     ChangeLog.md
 
+flag old-glibc
+  description: compile with post-2016 glibc, allowing atomic file writes
+  default:     False
+  manual:      False
```

Comment:
This should be `manual: True`, right? We don't want the dep solver trying to change this flag.